db/schema.rb

```diff
@@ -75,9 +76,9 @@
     t.datetime "created_at", precision: 6, null: false
     t.datetime "updated_at", precision: 6, null: false
     t.bigint "api_user_id"
-    t.index ["api_user_id"], name: "index_audits_on_api_user_id"
-    t.index ["planning_application_id"], name: "index_audits_on_planning_application_id"
-    t.index ["user_id"], name: "index_audits_on_user_id"
+    t.index ["api_user_id"], name: "ix_audits_on_api_user_id"
+    t.index ["planning_application_id"], name: "ix_audits_on_planning_application_id"
+    t.index ["user_id"], name: "ix_audits_on_user_id"
```

Comment:
I guess all the "ix_..._on_user_id" are because we couldn't fit the index name, although "index_something" would be a bit clearer about what they are doing I think. Still one only needs to read the whole line to understand so it matters not 🤷